versioned_docs/version-2.6/getting-started/installation-and-upgrade/installation-references/feature-flags.md

```diff
@@ -2,6 +2,10 @@
 title: Feature Flags
 ---
 
+<head>
+  <link rel="canonical" href="https://ranchermanager.docs.rancher.com/getting-started/installation-and-upgrade/install-upgrade-on-a-kubernetes-cluster/upgrades"/>
```

Comment:
```suggestion
  <link rel="canonical" href="https://ranchermanager.docs.rancher.com/getting-started/installation-and-upgrade/installation-references/feature-flags"/>
```

Reply:
Just pushed [b1d6243](https://github.com/rancher/rancher-docs/pull/621/commits/b1d62436c5df2ebb861fbded6d760f2822213aca) to correct the copy/paste error